swix/res/Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest.swixproj

```diff
@@ -1,42 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We don't need these files any more? Because we now set IsProductComponent?

Reply:
They were never used actually. I had them there since I was basing the loc after a different example which ended up being unnecessary.